VideoLocker/src/main/java/org/edx/mobile/view/MyCoursesListActivity.java

```diff
@@ -79,4 +118,34 @@ public void onFail(Exception ex) {
             logger.error(ex);
         }
     };
+
+    /**
+     * Event bus callback for new app version availability event.
+     *
+     * @param newVersionAvailableEvent The new app version availability event.
+     */
+    public void onEvent(final NewVersionAvailableEvent newVersionAvailableEvent) {
+        if (!newVersionAvailableEvent.isConsumed()) {
+            Snackbar snackbar = Snackbar.make(coordinatorLayout,
+                            newVersionAvailableEvent.getNotificationString(this),
+                            Snackbar.LENGTH_INDEFINITE)
+                    .setAction(R.string.app_version_upgrade_button,
+                            AppStoreUtils.OPEN_APP_IN_APP_STORE_CLICK_LISTENER)
+                    .setCallback(new Snackbar.Callback() {
+                        @Override
+                        public void onDismissed(Snackbar snackbar, int event) {
+                            newVersionAvailableEvent.markAsConsumed();
+                        }
+                    });
+            /* The Snackbar layout truncates the text to two lines, and the action
+             * text also takes up lots of space at the right. Manually increase
+             * the maximum lines limit to 10 in order to have our current messages
+             * displayed.
+             * TODO: Make the strings shorter?
```

Comment:
@1zaman I will take a look and tag you on the Version Upgrade Alerts wiki. Thanks!